xblockutils/base_test.py

```diff
@@ -89,7 +94,7 @@ def set_scenario_xml(xml):
         SCENARIOS.clear()
         add_xml_scenario("test", "Test Scenario", xml)
 
-    def go_to_view(self, view_name='student_view', student_id=None):
+    def go_to_view(self, view_name='student_view', student_id="student_1"):
```

Comment:
This change isn't necessarily backwards-compatible, since the new version will navigate to a different URL when student_id is omitted.  I don't know whether that matters.